src/pages/debrief/debrief.selector.ts

```diff
@@ -113,7 +113,7 @@ export const getVehicleCheckSeriousFaults =
     const competency: CommentedCompetency & MultiFaultAssignableCompetency = {
       comment: vehicleChecks.showMeTellMeComments || '',
       competencyIdentifier: CommentSource.VEHICLE_CHECKS,
-      competencyDisplayName: CommentSource.VEHICLE_CHECKS,
+      competencyDisplayName: 'Show Me/Tell Me',
```

Comment:
Could you please extract this into a constant similar to the CommentSource one? This label is used a few times in this file so it would be nice to avoid magic strings

Reply:
Yep good idea. Added a `CompetencyDisplayName` enum.